app/controllers/dams_resource_controller.rb

```diff
@@ -103,6 +103,18 @@ def rdf
     params[:xsl] = "normalize.xsl"
     data = get_html_data params, nil
     render :xml => data
+  end
+  def rdf_nt
```

Comment:
Hi Matt and Esme, 
The feature for content negotiation with adding .nt/.ttl extensions to the URL is implemented. See https://github.com/ucsdlib/damspas/commit/bbca23a86ecb927f70f6cf9bef4062619179d03d.